packages/dashboard/e2e/tests/utils/grid.ts

```diff
@@ -0,0 +1,134 @@
+import { Locator, Page } from '@playwright/test';
+import last from 'lodash/last';
```

Comment:
Consider avoiding taking on the dependency. `[].at(-1)` is supported natively by JS and is a safe operation (will not throw).